### PR TITLE
Typo fix in dependency config file for Ultra

### PR DESCRIPTION
**Pull request by laspsandoval**
# Change Summary

## Overview
Typo fix in dependency config file for Ultra

## Updated Files
- dependency_config_ultra.py
   - fixed typo

### CODE REVIEW

**Review by greglucas — APPROVED** [review]
Would it be possible to write this as:
```python
upstream_dependents = []
for dep in downstream_dependents:
    upstream_dep = PreProcessingDependency(
        primary_instrument=dep.dependent_instrument,
        primary_data_level=dep.dependent_data_level,
        primary_descriptor=dep.dependent_descriptor,
        dependent_instrument=dep.primary_instrument
        dependent_data_level=dep.primary_data_level
        dependent_descriptor=dep.primary_descriptor,
        direction="UPSTREAM",
    )
    upstream_dependents.append(upstream_dep)
```

Essentially reversing the directions. If so, perhaps we should add a method to `PreProcessingDependency` that does this for us automatically?

File: sds_data_manager/lambda_code/SDSCode/database/models.py
Comment (tech3371) [inline sds_data_manager/lambda_code/SDSCode/database/models.py:194]: I think it's ok to do this for this SIT but it won't be true after that when we add in inter-instrument dependency. Can you note that somewhere as reminder?

File: sds_data_manager/lambda_code/SDSCode/dependency_config_ultra.py
Comment (tech3371) [inline sds_data_manager/lambda_code/SDSCode/dependency_config_ultra.py:236]: may be add note here about what I was mentioning in first comment?

**Review by tech3371 — APPROVED** [review]
I commented on one concern I have of this change. But I like the reasons why we want to  do this.

File: sds_data_manager/lambda_code/SDSCode/database/models.py
Comment (tech3371) [reply sds_data_manager/lambda_code/SDSCode/database/models.py:194]: I thought of an idea for this case. I will share that after SDWG meeting.

Comment (tech3371) [reply sds_data_manager/lambda_code/SDSCode/database/models.py:194]: But I don't mean to not merge this. You can merge this because I will need to make same changes for other instruments. I can do that after this PR is merged.

**Pull request merged**
